src/ncp/spinel.c

```diff
@@ -1577,6 +1577,14 @@ const char *spinel_prop_key_to_cstr(spinel_prop_key_t prop_key)
         ret = "PROP_DATASET_DEST_ADDRESS";
         break;
 
+    case SPINEL_PROP_MESHCOP_JOINER_STATE:
+        ret = "PROP_MESHCOP_JOINER_STATE";
+        break;
+
+    case SPINEL_PROP_MESHCOP_JOINER_COMMISSIONING:
```

Comment:
`SPINEL_PROP_MESHCOP_JOINER_COMMISSIONING` -> `SPINEL_PROP_MESHCOP_JOINER_JOIN`?

Reply:
prefer to keep it as is - Here the wording `COMMISSIONING` is better in describe the thread commissioning process and as spinel is also part of wpantund which has wpanctl cmd `join` or other `JOIN_EXISTING` wording inside, the other purpose it to avoid possible confusion in understanding.

thoughts?